jmix-flowui/flowui/src/main/java/io/jmix/flowui/app/filter/condition/PropertyFilterConditionDetailView.java

```diff
@@ -108,9 +106,8 @@ protected void onReady(@SuppressWarnings("unused") ReadyEvent event) {
     }
 
     protected void initPropertyField() {
-        if (filterMetaClass != null) {
-            List<String> properties = filterMetadataTools.getPropertyPaths(filterMetaClass, query,
-                            propertiesFilterPredicate).stream()
+        if (currentConfiguration != null) {
```

Comment:
all of this solves some bug? I don't get how it relates to propertyHierarchyDepth